core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -185,6 +185,14 @@ abstract class TaskContext extends Serializable {
   @Evolving
   def resources(): Map[String, ResourceInformation]
 
+  /**
+   * (java-specific) Resources allocated to the task. The key is the resource name and the value
+   * is information about the resource. Please refer to
+   * [[org.apache.spark.resource.ResourceInformation]] for specifics.
+   */
+  @Evolving
+  def resourcesJMap(): java.util.Map[String, ResourceInformation]
```

Comment:
Ah, I missed that. We need MiMa exclusion.
```
[error]  * abstract method resourcesJMap()java.util.Map in class org.apache.spark.TaskContext is present only in current version
[error]    filter with: ProblemFilters.exclude[ReversedMissingMethodProblem]("org.apache.spark.TaskContext.resourcesJMap")
```

Reply:
I'll update shortly